src/Mod/Sketcher/Gui/DrawSketchHandlerArc.h

```diff
@@ -423,6 +432,21 @@ class DrawSketchHandlerArc: public DrawSketchHandlerArcBase
         }
     }
 
+    // Hint table structures
+    struct HintEntry
+    {
+        ConstructionMethod method;
+        SelectMode state;
+        std::list<Gui::InputHint> hints;
+    };
+
+    using HintTable = std::vector<HintEntry>;
+
+    // Static declaration
+    static const Gui::InputHint SWITCH_MODE_HINT;

```

Comment:
instead of such const you can for example define static function that will return proper hint.